src/utils/functions/index.jsx

```diff
@@ -0,0 +1,48 @@
+export const filterForecast = (array) => {
+  return array.filter((_, index) => index % 8 === 0);
+};
+
+const days = [
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday',
+  'Sunday',
+];
+
+const months = [
+  'Jan',
+  'Feb',
+  'Mar',
+  'Apr',
+  'May',
+  'Jun',
+  'Jul',
+  'Aug',
+  'Sep',
+  'Oct',
+  'Nov',
+  'Dec',
+];
+
+export const getTimeStamp = (unixTimeStamp) => {
+  const hours = new Date(unixTimeStamp * 1000).getHours();
+  const minutes = new Date(unixTimeStamp * 1000).getMinutes();
+  const twoDigitMinutes = minutes.toString().padStart(2, '0');
+  return `${hours}:${twoDigitMinutes}`;
+};
```

Comment:
tahle funkce by se měla jmenovat spíš něco jako "format date" ne?

Reply:
to by asi bylo lepsi to pojmenovat format date